contracts/bridges/BridgeTimelock.sol

```diff
@@ -0,0 +1,113 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "../ITimelock.sol";
+
+contract BridgeTimelock is ITimelock {
+    /** Custom errors **/
+    error BadDelay();
+    error BadETA();
+    error TransactionExpired();
+    error TransactionNotQueued();
+    error TransactionNotReady();
```

Comment:
better name than `TransactionNotReady`? `TransactionNotExecutable`?

Reply:
`TransactionEtaNotReached`?

Reply:
deleted this change to keep Timelock as similar as possible to its original version

Reply:
Fair enough, though I think personally think that, just like converting constants to immutables, custom errors are a safe improvement to make.